### PR TITLE
Updated all dependencies, now it works with latest versions of packages

### PR DESCRIPTION
**Pull request by dakorpar**
I wanted to use this package but it wasn't possible since my project is now up to date and this package is sadly outdated so I had conflicts. https://github.com/Kdyby/Aop/issues/32
Anyhow I decided to update all myself, I can't garantue that this works in all cases. I'm not fammiliar with this code and neither with PHPGenerator which changed a lot and introduced some breaking changes so I had to write our own from function on PointCutMethod, since newer versions of PHPGenerator don't do new static anymore but always new Method and then everything would fail. I just copied code from new PHPGenerator factory and made it work here. It's far from perfect but at this point it works and I'm ok with it. Sorry for 5 commits I was struggling to make it work and don't have nerves anymore to squuash them, it's easier on your end anyway... :)

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -17,37 +17,36 @@
...
+		"nette/caching": "~2.4",
+		"nette/component-model": "~2.3",
+		"nette/database": "~2.4",
+		"nette/di": "~2.4",
+		"nette/finder": "~2.4",
+		"nette/forms": "~2.4",
+		"nette/http": "~2.4",
+		"nette/mail": "~2.4",
+		"nette/neon": "~2.4",
+		"nette/php-generator": "2.6.2",
```
Comment (Majkl578) [inline composer.json]: Why fixed version?

Comment (dakorpar) [inline composer.json]: I was playing with it and checking on which version will it fail, later i put it on newest while testing but forgot to remove it. In commit bellow this is removed.

**Comment by dakorpar** [comment]
If anyone need new working version with latest dependencies able to run on updated project can use my fork: 
```
	"repositories": [
		{
			"type": "vcs",
			"url": "https://github.com/dakorpar/aop"
		}
	]
```
I'm using it on big project and allways keeping all libs up to date.

**Comment by dakorpar** [comment]
moved to #34

**Pull request closed without merging**